docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -14,7 +14,8 @@ A data node also holds various properties and attributes accessible through the
 -   _**id**_: The unique identifier of this data node.
 -   _**name**_: The user-readable name of the data node.
 -   _**owner_id**_: The identifier of the owner (pipeline_id, scenario_id, cycle_id) or `None`.
--   _**last_edit_date**_: The date and time of the last modification.
+-   _**last_edit_date**_: The date and time of the last modification either by Taipy execution or
+    an external factor.
```

Comment:
external... process?
Factor sounds awkward to me.

Reply:
@FabienLelaquais meaning the modification can be done by anyone, users or another script, etc. so I'm not sure process is the right one here

Reply:
I understand, sure.
If you feel comfortable with factor, please go ahead. I don't, but nothing better to propose though.

Reply:
@jrobinAV 
I'm not sure because a user can directly make changes to the file, and I think component doesn't cover it

Reply:
Right